index.html

```diff
@@ -37,154 +36,213 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
-               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
-               <ul class ="lang">
-                   <li class="language">html</li>
-                   <li class="language">css</li>
-                   <li class="language">javascript</li>
-                   <li class="language">Ruby on rails</li>
-               </ul>
-               <div class="butt"><button class="btn" type="button"> See Project</button></div>
-           </article>
            
-           <article class="project">
-               <div class ="image-placeholder"></div>
-               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
-               <ul class ="lang">
-                   <li class="language">html</li>
-                   <li class="language">css</li>
-                   <li class="language">javascript</li>
-                   <li class="language">Ruby on rails</li>
-               </ul>
-               <div class="butt"><button class="btn" type="button"> See Project</button></div>
-           </article> 
-       </div>
-           <div class="design-3">
-           <article class="project">
-               <div class ="image-placeholder"></div>
-               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
-               <ul class ="lang">
-                   <li class="language">html</li>
-                   <li class="language">css</li>
-                   <li class="language">javascript</li>
-                   <li class="language">Ruby on rails</li>
-               </ul>
-               <div class="butt"><button class="btn" type="button"> See Project</button></div>
-           </article>
-            <article class="project">
-               <div class ="image-placeholder"></div>
-               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
-               <ul class ="lang">
-                   <li class="language">html</li>
-                   <li class="language">css</li>
-                   <li class="language">javascript</li>
-                   <li class="language">Ruby on rails</li>
-               </ul>
-               <div class="butt"><button class="btn" type="button"> See Project</button></div>
-           </article>
-           </div>
-           </div>
-       </div>
-       </section>
+           
+            <section class = "work-section">
+             <h2 class ="recent" >My Recent Works</h2>
+             <hr class="line">
+            
+            <div class ="project-container">
+                <div class="design-1">
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
```

Comment:
- Good job done so far 👍 , but there are some differences between your design and the given Figma design. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this for all project cards in your mobile and desktop versions 👍 👍 

Your project:
- Please use the border-radius property on each of your technology list
<img width="221" alt="teheththh" src="https://user-images.githubusercontent.com/73607512/206674507-46800098-6f8b-46e5-a777-ea484cf3af45.PNG">

Figma Design:
<img width="123" alt="gr3gg" src="https://user-images.githubusercontent.com/73607512/206674680-c0807f8e-734e-4819-9cd5-5c0955c7cff5.PNG">